pkg/my/domain.go

```diff
@@ -27,7 +29,9 @@ func Domains(client *node.Client, nickname string) ([]string, error) {
 
 	domains := []string{}
 
-	domainsEntry, err := node.DatastoreEntry(client, dns.DNSRawAddress, ownedPrefix+wallet.Address)
+	keyWithHeader := convert.EncodeStringUint32ToUTF8(ownedPrefix + wallet.Address)
```

Comment:
Can you describe what the function is doing ?
Can you then rename this variable by choosing a name that match hte function scope and with having useless information ?